aii-core/src/test/perl/NCM/Component/PXELINUX/constants.pm

```diff
@@ -0,0 +1 @@
+../../../../../../../aii-pxelinux/target/lib/perl/NCM/Component/PXELINUX/constants.pm
```

Comment:
ok, but if you wan to test aii-core directly, you need to test pxelinux first. if a symlink to src works, i'd go for that. otherwise, others will find it out 😄